platform-api/src/dependencies.py

```
@@ -207,21 +207,77 @@ async def get_sanity(settings: WorkerSettings = Depends(get_settings)) -> Sanity
+            JOIN user ON session.user_id = user.id
+            WHERE session.token = ?
+              {role_clause}
+              AND session.expires_at > (unixepoch() * 1000)
+        """
+        binds = [token, require_role] if require_role else [token]
+
+        try:
+            result = await db.prepare(query).bind(*binds).first()
+            if not result:
+                return None
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)